source/MaterialXTest/ShaderGen.cpp

```diff
@@ -1008,6 +1008,104 @@ TEST_CASE("Subgraphs", "[shadergen]")
     }
 }
 
+TEST_CASE("Materials", "[shadergen]")
+{
+    mx::DocumentPtr doc = mx::createDocument();
+
+    mx::FilePath searchPath = mx::FilePath::getCurrentPath() / mx::FilePath("documents/Libraries");
+    loadLibraries({ "stdlib", "sxpbrlib" }, searchPath, doc);
+
+    std::vector<mx::MaterialPtr> materials;
+
+    mx::MaterialPtr material1 = doc->addMaterial("example1");
+    material1->addShaderRef("surface", "standardsurface");
+    materials.push_back(material1);
```

Comment:
New test for materials, just a simple material with a single standardsurface for now.